extensions/git/src/git.ts

```diff
@@ -503,7 +512,9 @@ export interface Commit {
 	hash: string;
 	message: string;
 	parents: string[];
-	authorEmail?: string | undefined;
+	authorDate?: Date;
+	authorName?: string;
+	authorEmail?: string;
```

Comment:
Cool, I'll wait for this to come in before merging https://github.com/microsoft/vscode/pull/89005.

Reply:
The merge probably wouldn't go very well, since I changed the parsing from how it was. But I can easily add `commitDate` in if we want.